.github/workflows/precommit.yaml

```diff
@@ -0,0 +1,21 @@
+name: pre-commit
+
+on:
+  pull_request:
+  push:
+    branches: [master]
+
+jobs:
+  pre-commit:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-python@v2
```

Comment:
Could you explicitely set a Python version here ? (Maybe several ones with `strategy.matrix`)